src/ReactiveUI/ReactiveBinding.cs

```diff
@@ -76,17 +72,14 @@ internal class ReactiveBinding<TView, TViewModel, TValue> : IReactiveBinding<TVi
     {
         private IDisposable bindingDisposable;
 
-        /// <summary>
```

Comment:
This comment doesn't really add anything, and we're in an internal class so not sure it really needs it - the same could be set for the property comments below, but at least they're not actually wrong...